packages/prisma/schema.prisma

```diff
@@ -204,6 +204,8 @@ model User {
   verifiedNumbers      VerifiedNumber[]
   hosts                Host[]
 
+  @@index([username])
+  @@index([allowDynamicBooking])
```

Comment:
is this needed?

Reply:
Some queries are filtering based on these fields so indexes should speed that up? I'll be honest though I'm not very familiar with db optimizations with indexes though

Reply:
Let's leave these out, we should add indexes but we should also evaluate our query optimisation before we do so.